asklib.module

```diff
@@ -124,6 +124,43 @@ function asklib_cron() {
       $question->save();
     }
   }
+
+  // Clear contact information from 2 year old questions every day after 1 am.
```

Comment:
```suggestion
  // Clear contact information of questions older than 2 years every day after 1 am.
```

It could also make sense to split some of this info to be closer to the mktime() calls.